WcaOnRails/app/views/competitions/_results_table.html.erb

```diff
@@ -41,7 +41,7 @@
       <tr class="sort-by-<%= result.format.sort_by.to_s %> <%= result.muted ? "text-muted" : "" %>">
         <% if !hide_event %>
           <td class="event">
-            <% if !result.muted && result.eventId != results[i - 1].eventId %>
+            <% if !result.muted && (i == 0 || result.eventId != results[i - 1].eventId) %>
```

Comment:
Nit: it might be a little cleaner to split this out into a separate, "previous event id" variable. Your call!

Reply:
May be cleaner, but in the block is quite huge, so I don't quite like declaring a variable above and updating it way later in the code ^^